shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/carmin/security/CarminDatasetProcessingSecurityService.java

```diff
@@ -0,0 +1,80 @@
+package org.shanoir.ng.processing.carmin.security;
+
+import org.shanoir.ng.processing.carmin.model.CarminDatasetProcessing;
+import org.shanoir.ng.shared.model.Study;
+import org.shanoir.ng.study.rights.StudyRightsService;
+import org.shanoir.ng.utils.KeycloakUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Service
+public class CarminDatasetProcessingSecurityService {
+
+    private static final String ROLE_ADMIN = "ROLE_ADMIN";
```

Comment:
Hi Alae, if you declare the constant, it would be great using it below, or not declaring it. Thank you and with kind regards, Michael